python/cac_tripplanner/cac_tripplanner/views.py

```diff
@@ -1,10 +1,9 @@
 from django.shortcuts import render_to_response
+from django.template import RequestContext
+from django.conf import settings
 
 def home(request):
-    return render_to_response('home.html')
+    return render_to_response('home.html', dict(debug=settings.DEBUG))
```

Comment:
Ah, here it is:
https://docs.djangoproject.com/en/1.7/ref/templates/api/#django-core-context-processors-debug